contentctl/objects/constants.py

```diff
@@ -136,4 +136,11 @@
 RBA_OBSERVABLE_ROLE_MAPPING = {
     "Attacker": 0,
     "Victim": 1
-}
\ No newline at end of file
+}
+
+# TODO (cmcginley): do these belong here? or in inspect.py?
+ESCU_APP_UID = 3449
```

Comment:
Do these "constants" belong in this file? Or are they too specific and should live in `config.py`?

Reply:
Good point.

Agreed that we want to make this as generic as possible. That said, do we want the default "download previous build from Splunkbase" behavior to support 3rd party workflows too? Do we expect any other users to be publishing content bundles to Splunkbase?

If not, then at least `ESCU_APP_UID` is a constant in my mind and not a config value. If yes, then yeah it should be generalized for any app UID and pulled from `config.app.uid`.

Reply:
On second thought, this is going to require a lot more thought.
There are two possibilities for a customer:
1. I am building and deploying a private app.  It is not on splunkbase, so I need to provide a path to this file (instead of downloading it from Splunkbase).
2. I am building and deploying an app that is published to Splunkbase. In this case, the downloader can work the same for this as it does for ESCU.

If we want/need to support both use cases, then `contentctl inspect` would need another (optional?) argument with the direct path to a package.

I am marking this as resolved, but have created the following issue for tracking:

Reply:
Unresolving this bc we already support the possibility of specifying a local path to a build (see `previous_build` in the `inspect` config).

Thoughts @pyth0n1c

Reply:
Still leaving this one open. If you refer to my comment:

I think it's a one line change to get the parsing working for non ESCU apps.

Reply:
@pyth0n1c see https://github.com/splunk/contentctl/pull/280#discussion_r1761565201

Reply:
I approved the resolved the linked change in the comment above, so I am approving this one as well!